IOS/SwiftSeedProject/SwiftSeedProject/Info.plist

```diff
@@ -4,6 +4,8 @@
 <dict>
 	<key>Base URL</key>
 	<string>$(BASE_URL)</string>
+	<key>API Key</key>
```

Comment:
Can you add something more to this name? Something like `newsapi API key` or ``news API key`

(We can easily have multiple api keys)